src/kEvent.mli

```diff
@@ -75,7 +75,7 @@ val log_counterexample : Lib.kind_module -> Lib.log_level -> string list -> Tran
 
     Should only be used by the invariant manager, other modules must use
     {!prop_status} to send it as a message. *)
-val log_prop_status : Lib.log_level -> TransSys.t -> (string * Property.prop_status) list -> unit
+val log_prop_status : Lib.log_level -> TransSys.t -> (string * Property.prop_status) list -> (string * Property.prop_kind) list -> unit
```

Comment:
The new argument is related to the third argument. Instead of adding a new argument to pass the list of property kinds separately from the list of status, I would modified the current argument to pass all info together: ` (string * Property.prop_status * Property.prop_kind) list`. In other words, I would combine the two lists outside of this function instead of within the function. This prevents calls to this function where the lengths of the two lists do not match.